### PR TITLE
(DI-3260) Adding agentless windows task (for choco)

### PR DESCRIPTION
**Pull request by HairyMike**

### CODE REVIEW

File: spec/acceptance/windows_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# run a test task
+require 'spec_helper_acceptance'
```
Comment (eoinmcq) [inline spec/acceptance/windows_spec.rb]: Should you have any failure cases in here?

Also how do you run these? Asking for a friend...

Comment (HairyMike) [reply spec/acceptance/windows_spec.rb]: Hi @eoinmcq,

I'll defer these to @tphoney :)

**Comment by tphoney** [comment]
Please do not merge, without understanding what you are doing. This has added to our headaches !!!!

**Comment by tphoney** [comment]
image

**Pull request merged**